### PR TITLE
Fix/66421 add tickets filter

### PR DESCRIPTION
**Pull request by elimn**
Ticket: https://central.tri.be/issues/66421

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Comment refers to a TODO task &quot;Remove in 4.6. This only exists because version =&gt;4.5.10 of TEC runs apply_filter() instead of calling this method directly&quot; | [Tickets.php:428](https://github.com/moderntribe/event-tickets/blob/b5bd15159a7c1796886e56d27e9c7393de0292ce/src/Tribe/Tickets.php#L428)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Comment refers to a TODO task &quot;Remove in 4.6. This only exists because version =&lt;4.5.10 of TEC runs apply_filter() instead of calling this method directly&quot; | [Tickets.php:428](https://github.com/moderntribe/event-tickets/blob/1aa321ff3ca273a9f0e9b36c835e2721ae27110d/src/Tribe/Tickets.php#L428)

> via jenkins/codesniffer-phpcs

File: readme.txt [hunk 1]
```diff
@@ -242,6 +242,7 @@ The plugin is produced by [Modern Tribe Inc](http://m.tri.be/18uc).
 * Fix - Added "View Tickets" link to Custom Post Types when appropriate. Thank you @19ideas for helping identify this. [67570]
 * Fix - Fix some layout issues with the "Email Attendees" modal in the Attendees list admin view, especially when viewed on phones or tablets. Props to @event-control for reporting this! [80975]
 * Fix - Avoid notice-level errors when calling ticket stock functions in relation to events with unlimited stock (props to Lou Anne for highlighting this) [78685]
+* Tweak - Documenter filter for available Ticket Modules, and used its method instead more places[66421]
```
Comment (borkweb) [inline readme.txt]: `Document` vs `Documenter`? Add a space between the last word and the ticket?

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Comment refers to a TODO task &quot;Remove in 4.6. This only exists because version =&lt;4.5.10 of TEC runs apply_filter() instead of calling this method directly&quot; | [Tickets.php:428](https://github.com/moderntribe/event-tickets/blob/0e3070708a7e5fb658483d95bcf65630d976d3fe/src/Tribe/Tickets.php#L428)

> via jenkins/codesniffer-phpcs

**Comment by elimn** [comment]
I need to write a bot that automatically corrects my typos!

Fixed. @borkweb

**Pull request merged**